pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,183 @@
+# Running Buildkite Agent on Google Cloud Platform
+
+The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). You may like to run the agent using individual VM instances on Google Compute Engine for fine control over long–lived agents. Or you may like to run a scalable cluster of agents using Docker and Kubernetes on the Google Container Engine to perform Docker–based builds.
```

Comment:
Would be good to swap this around a bit - the second and third sentences might work better with the 'why' before the 'what':

> For fine control over long-lived agents, you can run the agent using individual VM instances.